### ISSUE TITLE
how to compute val loss during training

### ISSUE THREAD

**Issue by chiheb10** [xdever/RFCN-tensorflow#48]
hi, how can I add the computation of validation loss during training
